src/manual-multiple-soulmates/web/index.html

```diff
@@ -56,46 +56,56 @@ <h2 class="blink__anchor advert__title">[%Offer3Title%]</h2>
                     </span>
                 </div>
             </a>
+            <a class="blink advert advert--manual advert--soulmate hide-until-tablet" href="%%CLICK_URL_UNESC%%[%Offer4URL%]" data-link-name="Offer 4 | [%Offer4Title%]" target="_top">
```

Comment:
Right. Then I suppose you can remove `Offer4LinkText` in `cleanupButtons`, can't you?